src/types.ts

```diff
@@ -6,10 +6,15 @@ export type Context = Record<string, ContextHTTPVars>;
 
 export type Presentation = 'lines' | 'sum' | 'average' | 'min' | 'max';
 
+export interface Selections {
+  context?: Record<string, unknown>;
+}
+
 export interface QueryParams {
```

Comment:
Not your fault, but an Interface where everything is optional makes me nervous. Do we know when we need what? Or if some things have defaults?

Reply:
To be honest it's just defensive programming. As we didn't have a typing error without this, we basically don't know anything about it. This is why I choose to make it optional. I think we will have to have some refactoring in order to be sure about this?!